src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/provisioner/main.py

```diff
@@ -53,10 +61,32 @@ def create_or_update_account(org_session, account, adf_role_name, account_id=Non
         ), 'delete_default_vpc'
     )
 
-    LOGGER.info(f'Ensuring account {account_id} (alias {account.alias}) is in OU {account.ou_path}')
+    LOGGER.info(
+        f'Ensuring account {account_id} (alias {account.alias}) is in OU {account.ou_path}')
     org_session.move_account(account_id, account.ou_path)
     if account.delete_default_vpc:
         ec2_client = role.client('ec2')
+        all_regions = get_all_regions(ec2_client)
+        args = (
+            (account_id, region, role)
+            for region in all_regions
+        )
+        with ThreadPoolExecutor(max_workers=10) as executor:
+            for _ in executor.map(lambda f: schedule_delete_default_vpc(*f), args):
+                pass
+
+    LOGGER.info(f'Ensuring account alias for {account_id} of {account.alias}')
+    org_session.create_account_alias(account.alias, role)
+
+    if account.tags:
+        LOGGER.info(
+            f'Ensuring tags exist for account {account_id}: {account.tags}')
+        org_session.create_account_tags(account_id, account.tags)
+
+
+@tenacity.retry(wait=tenacity.wait_fixed(5), stop=tenacity.stop_after_attempt(60))
```

Comment:
This should not use fixed wait times. The risk is that all threads get notified to back-off, after which all of them wait a fixed amount of time, then retry and fail again. As is further explained here [0].

I would recommend to update the `wait` argument to: `tenacity.wait_random_exponential()`

[0] https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/